pipeline/unified/build-unsigned-release-packages.yaml

```diff
@@ -45,6 +47,24 @@ steps:
     - script: yarn build:unified:all --unified-version=$(Build.BuildNumber) --unified-canary-instrumentation-key=$(UnifiedCanaryInstrumentationKey) --unified-insider-instrumentation-key=$(UnifiedInsiderInstrumentationKey) --unified-prod-instrumentation-key=$(UnifiedProdInstrumentationKey)
       displayName: yarn build:unified:all
 
+    - script: |
+          security create-keychain -p password $(System.DefaultWorkingDirectory)/agent.keychain
+          security default-keychain -s $(System.DefaultWorkingDirectory)/agent.keychain
+          security unlock-keychain -p password $(System.DefaultWorkingDirectory)/agent.keychain
+          echo "$(mac-notarization-developerid-certificate-base64p12)" | base64 -D > $(System.DefaultWorkingDirectory)/cert.p12
+          security import $(System.DefaultWorkingDirectory)/cert.p12 -k $(System.DefaultWorkingDirectory)/agent.keychain -P "$(mac-notarization-developerid-certificate-password)" -T /usr/bin/codesign
+          security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k $(System.DefaultWorkingDirectory)/agent.keychain
+          yarn unified-mac-sign $(mac-notarization-developerid-certificate-identity) $(System.DefaultWorkingDirectory)/src/electron/resources/entitlements.plist
+      displayName: Signing app bundle
```

Comment:
It's a bit confusing that the build named "build-unsigned-release-packages" now contains a signing step. Would it be possible to adjust the naming (maybe just "build-release-packages") accordingly?

Reply:
It would also be nice to leave a comment here explaining the difference between this "code sign" step vs the other "code sign" step in the signing build

Reply:
I've changed the names for the tasks to hopefully describe what's happening more accurately.

wrt renaming the step: I agree but I think it would make sense to rename the pipelines and such as well and figured I can do that in a future rename PR (there's a handful of files that'll be touched).